crawler.go

```diff
@@ -130,111 +127,35 @@ func (c crawler) navegacaoSite(ctx context.Context, xpath string) error {
-		4:  "Abr",
-		5:  "Mai",
-		6:  "Jun",
-		7:  "Jul",
-		8:  "Ago",
-		9:  "Set",
-		10: "Out",
-		11: "Nov",
-		12: "Dez",
-	}
-	c.month = convMap[monthConverted]
-
-	var selectYear, selectMonth string
-	if tipo == "contracheque" {
-		selectYear = `/html/body/div[5]/div/div[4]/div[2]/div/div[1]/div[5]/div/div[1]/div[1]`
-		selectMonth = `/html/body/div[5]/div/div[4]/div[2]/div/div[1]/div[5]/div/div[4]/div[1]`
-	} else {
-		selectYear = `/html/body/div[5]/div/div[1]/div[2]/div/div[1]/div[5]/div/div[1]`
-		selectMonth = `/html/body/div[5]/div/div[1]/div[2]/div/div[1]/div[5]/div/div[4]`
-	}
-
-	year := fmt.Sprintf("//*[@title='%s']", c.year)
-	month := fmt.Sprintf("//*[@title='%s']", c.month)
-
-	//Tenta pegar o mês que está selecionado atualmente lá no site do MPMS
-	selectedMonth, err := c.getSelectedMonth(ctx, tipo)
-	if err != nil {
-		log.Fatalf("erro ao obter mês selecionado no site: %v", err)
-	}
-
-	/*
-		Quando o mês selecionado lá no site for igual ao mês que queremos coletar,
-		selecionamos apenas o ano da coleta. Isso resolve a diferença entre a
-		quantidade de clicks, de quando queremos coletar um mês diferente do que já
-		está selecionado lá no site, e a quantidade de clicks de quando queremos fazer
-		a coleta do mês que já está selecionado.
-	*/
-	if selectedMonth == c.month {
-		//Faz a seleção apenas do ano
-		return chromedp.Run(ctx,
-			// Seleciona ano
-			chromedp.Click(selectYear, chromedp.BySearch, chromedp.NodeReady),
-			chromedp.Sleep(c.timeBetweenSteps),
-
-			chromedp.Click(year, chromedp.BySearch, chromedp.NodeReady),
-			chromedp.Sleep(c.timeBetweenSteps),
-
-			chromedp.Click(selectYear, chromedp.BySearch, chromedp.NodeReady),
-			chromedp.Sleep(c.timeBetweenSteps),
-
-			chromedp.Click(year, chromedp.BySearch, chromedp.NodeReady),
-			chromedp.Sleep(c.timeBetweenSteps),
-		)
-	} else {
-		//Faz a seleção do ano e do mês
-		return chromedp.Run(ctx,
-			// Seleciona ano
-			chromedp.Click(selectYear, chromedp.BySearch, chromedp.NodeReady),
-			chromedp.Sleep(c.timeBetweenSteps),
-
-			chromedp.Click(year, chromedp.BySearch, chromedp.NodeReady),
-			chromedp.Sleep(c.timeBetweenSteps),
-
-			chromedp.Click(selectYear, chromedp.BySearch, chromedp.NodeReady),
-			chromedp.Sleep(c.timeBetweenSteps),
-
-			chromedp.Click(year, chromedp.BySearch, chromedp.NodeReady),
-			chromedp.Sleep(c.timeBetweenSteps),
-
-			// Seleciona mês
-			chromedp.Click(selectMonth, chromedp.BySearch, chromedp.NodeReady),
-			chromedp.Sleep(c.timeBetweenSteps),
-
-			chromedp.Click(month, chromedp.BySearch, chromedp.NodeReady),
-			chromedp.Sleep(c.timeBetweenSteps),
-		)
-	}
-}
-
-// exportaPlanilha clica no botão correto para exportar para excel, espera um tempo para download renomeia o arquivo.
-func (c crawler) exportaPlanilha(ctx context.Context, fName string) error {
-	tctx, tcancel := context.WithTimeout(ctx, 30*time.Second)
-	defer tcancel()
+	var rows []*cdp.Node
+	chromedp.Run(ctx, chromedp.Nodes("tr", &rows, chromedp.ByQueryAll))
```

Comment:
Verificar erro retornado pelo método chromedp.Run. Se o erro for diferente de nil, retornar (olhar melhoria no texto do erro que mencionei acima).